lib/daru/dataframe.rb

```diff
@@ -1817,10 +1817,14 @@ def method_missing(name, *args, &block)
       elsif has_vector? name
         self[name]
       else
-        super(name, *args, &block)
+        super
```

Comment:
@v0dro `super` without arguments is called implicitly with _exactly_ the arguments of current method